torchinfo/layer_info.py

```diff
@@ -263,7 +263,7 @@ def check_recursive(self, layer_ids: set[int]) -> None:
         If the current module is already-used, mark as (recursive).
         Must check before adding line to the summary.
         """
-        if any(self.module.named_parameters()) and self.layer_id in layer_ids:
```

Comment:
I'm surprised we can safely remove this. Unfortunately I don't remember why this was needed in the first place :/